packages/mc-http-server/bin/start-mc.js

```diff
@@ -10,7 +10,7 @@ const { compileHtml } = require('@commercetools-frontend/mc-html-template');
 const {
   packageLocation: applicationStaticAssetsPath,
 } = require('@commercetools-frontend/assets');
-const startServer = require('../server');
+const launchServer = require('../server');
```

Comment:
Anywhere else we name the export `launchServer` whereas start is the `listen` callback.